draft-ietf-tls-tls13.md

```diff
@@ -4802,21 +4802,23 @@ by IANA:
 This section provides a summary of the legal state transitions for the
 client and server handshakes.  State names (in all capitals, e.g.,
 START) have no formal meaning but are provided for ease of
-comprehension.  Messages which are sent only sometimes are indicated
-in [].
+comprehension.  Actions which are taken only in certain circumstances are
+indicated in [].
 
 ## Client
 
 ~~~~
                            START <----+
             Send ClientHello |        | Recv HelloRetryRequest
-         /                   v        |
-        |                  WAIT_SH ---+
-    Can |                    | Recv ServerHello
-   send |                    V
-  early |                 WAIT_EE
-   data |                    | Recv EncryptedExtensions
-        |           +--------+--------+
+   [Rekey out to early keys] |        | Send ClientHello
```

Comment:
Does this extra 'Send ClientHello' before returning to START belong here, when it's already going to retry with a second ClientHello after returning to START?